src/sync_bmc_data_ifaces.cpp

```diff
@@ -54,6 +54,10 @@ sdbusplus::async::task<>
 bool SyncBMCDataIface::set_property([[maybe_unused]] disable_sync_t type,
                                     bool disable)
 {
+    if (sync_events_health_ == SyncEventsHealth::Ok && disable)
```

Comment:
I think we could move line 67 to avoid checking every time the system or user tries to update the disable property.

Also, if the sync event health is Paused and sync is enabled, we should change it to Ok, right? From what I see, it currently only changes after a full sync, but we don’t need to wait for that here, do we? If the full sync fails, the health will be set to Critical anyway, right?